### PR TITLE
fix(OnyxNavAppArea): max-height not working correctly

### PR DESCRIPTION
**Pull request by JoCa96**
Relates to #1266

### CODE REVIEW

**Review by BoppLi — DISMISSED** [review]
doesn't this need a `patch` changeset?

**Comment by JoCa96** [comment]
> doesn't this need a `patch` changeset?
@BoppLi done

File: packages/sit-onyx/src/components/OnyxNavAppArea/OnyxNavAppArea.vue [hunk 1]
```diff
@@ -61,7 +61,7 @@ const buttonLabel = computed(() => props.label ?? t.value("navigation.goToHome")
     padding: var(--onyx-spacing-md);
     font-weight: 600;
     white-space: pre-line;
-    max-height: 100%;
+    max-height: 100cqh;
```
Comment (larsrickert) [inline packages/sit-onyx/src/components/OnyxNavAppArea/OnyxNavAppArea.vue:64]: Fancy^^ Never saw this unit before :D
A comment might help if we take a look at this in the future or we just set the logo max width also to 1.5rem

**Review by BoppLi — DISMISSED** [review]

File: packages/sit-onyx/src/components/OnyxNavAppArea/OnyxNavAppArea.vue (hunk 1 above)
Comment (JoCa96) [reply packages/sit-onyx/src/components/OnyxNavAppArea/OnyxNavAppArea.vue:65]: added comment

**Review by BoppLi — APPROVED** [review]

**Pull request merged**